src/Datagrid/SimplePager.php

```diff
@@ -128,6 +128,7 @@ public function getResults($hydrationMode = null)
             return $this->results;
         }
 
+        // @phpstan-ignore-next-line
         $this->results = $this->getQuery()->execute([], $hydrationMode);
```

Comment:
I think a `NEXT_MAJOR` version should be added to remove those arguments.

Reply:
The method is already deprecated and removed in the next major.